docs/Installation-Guide.rst

```diff
@@ -13,6 +13,21 @@ Users who want to perform benchmarking can make LightGBM output time costs for d
 
 It is possible to build LightGBM in debug mode. In this mode all compiler optimizations are disabled and LightGBM performs more checks internally. To enable debug mode you can add ``-DUSE_DEBUG=ON`` to CMake flags or choose ``Debug_*`` configuration (e.g. ``Debug_DLL``, ``Debug_mpi``) in Visual Studio depending on how you are building LightGBM.
 
+.. _sanitizers:
+
+In addition to the debug mode, LightGBM can be built with compiler sanitizers.
+To enable them add ``-DUSE_SANITIZER=ON -DENABLED_SANITIZERS="san_1;san_2;...;san_n"`` to CMake flags.
+``san_1`` ... ``san_n`` can refer to the following supported sanitizers:
```

Comment:
I was confused by this notation. I think it would be easier to use if we made this documentation copy-pastable (using the actual flag names).

```suggestion
To enable them add ``-DUSE_SANITIZER=ON -DENABLED_SANITIZERS="address;leak;undefined;thread"`` to CMake flags.
These values refer to the following supported sanitizers:
```